src/BloomBrowserUI/bookEdit/EditViewFrame.jade

```diff
@@ -18,7 +18,7 @@ html
 			$(document).ready(function () {
 				// The iframe communication channel needs to be initialized before the child documents are loaded.
 				interIframeChannel = new interIframeChannel();
-				document.getElementById('page').src = '/bloom/currentPageContent';
+				document.getElementById('page').src = 'insertFakeFileHere';
```

Comment:
Probably we'd get some clarity by making it look like a template by surrounding in {}, as in "{pageFilePath}".

Then, maybe a brief comment about who fills it in? Reading it, I wondered between the trade off between filling in a template and instead having a constant that our server detects. E.g., if the server saw /bloom/CurrentPageContent, why couldn't it return the same thing?